.github/workflows/build.yml

```diff
@@ -290,9 +310,7 @@ jobs:
 
     - name: Prepare Upload
       shell: bash
-      run: |
-        find . -type f -name 'makoureactor-continuous-*-Debug*' -delete
-        find . -type f -name 'makoureactor-continuous-*' -exec bash -c 'mv "{}" $(echo {} | sed 's/continuous-//' | sed 's/-Release//')' ';'
+      run: find . -type f -name 'makoureactor-continuous-*' -exec bash -c 'mv "{}" $(echo {} | sed 's/continuous-//')' ';'
```

Comment:
If you remove the `sed 's/-Release//'`, you will have -Release in the uploaded file. 
Since you also removed the Debug build, I propose to remove the build_type parameter from the matrix completely

Reply:
Fair enough, let me do that